spec/syntax.md

```diff
@@ -733,7 +733,13 @@ A **_<dfn>name</dfn>_** is a character sequence used in an _identifier_
 or as the name for a _variable_
 or the value of an _unquoted literal_.
 
-_Variable_ names are prefixed with `$`.
+A _name_ can be preceded or followed by bidirectional marks or isolating controls
+to aid in presenting names that contain right-to-left or neutral characters.
+These characters are **not** part of the _name_ and MUST be treated as if they were not present
+when matching _name_ or _identifier_ strings or _unquoted literal_ values.
+Implementations MAY remove these characters from a _message_.
```

Comment:
What does this mean? When/why/how would these characters be removed?

Reply:
For example, an implementation might remove the strong marks from around names when storing the names (in order to facilitate faster comparison). An implementation might also remove them when serializing, at the cost of some legibility.

I could just get rid of this line.

Reply:
I think we're already permitting that with the preceding sentence, so removing this would be clearer.

```suggestion
```